eval_with_modal.py

```
@@ -0,0 +1,187 @@
+    ignore=IGNORE_PATTERNS
+).copy_local_file("pyproject.toml", "pyproject.toml").copy_local_file("poetry.lock", "poetry.lock").env({"ENV": os.getenv('ENV'), "LLM_API_KEY": os.getenv("LLM_API_KEY")}).poetry_install_from_file(poetry_pyproject_toml="pyproject.toml")
+
+
+@app.function(
+    image=image,
+    gpu="T4",
+    concurrency_limit=5
+)
+async def run_single_repo(instance_data: dict, disable_cognee: bool = False):
+    import os
+    import json
```

Ruff (0.8.2)

Redefinition of unused `json` from line 6

Remove definition: `json`

(F811)